backend/EasySpeak.Core/EasySpeak.Core.DAL/Context/Seeder.cs

```diff
@@ -168,15 +171,14 @@ private static IList<Lesson> GenerateLessons(int count = 10)
             .Generate(count);
     }
 
-    private static IList<Tag> GenerateTags(int count = 20)
+    private static IList<Tag> GenerateTags()
     {
-        Faker.GlobalUniqueIndex = 0;
-
-        return new Faker<Tag>()
-            .UseSeed(10)
-            .RuleFor(t => t.Id, f => f.IndexGlobal)
-            .RuleFor(t => t.Name, f => f.Random.Word())
-            .Generate(count);
+        return AllTags.Select((t, i) => new Tag
+        {
+            Id = ++i,
+            Name = t,
+            CreatedAt = DateTime.Now
```

Comment:
Let's start Id with 0, please
CreatedAt - use DefaultDate or we will change this data on every migration

Reply:
I've intentionally started Id with 1, because I get this error if start with 0
`The seed entity for entity type 'Tag' cannot be added because a non-zero value is required for property 'Id'. Consider providing a negative value to avoid collisions with non-seed data.`

Reply:
`CreatedAt ` changed and works fine

Reply:
Ah, ok, id makes sense